scapy/arch/bpf/core.py

```diff
@@ -159,22 +180,24 @@ def get_working_ifaces():
         if ifflags & 0x1:  # IFF_UP
 
             # Get a BPF handle
-            fd = get_dev_bpf()[0]
-            if fd is None:
+            file_desc = get_dev_bpf()[0]
```

Comment:
As I have already stated, we do not accept pylint / flake8 fixes (except in very specific conditions which include adding tests to prevent regressions). In this situation, your patch breaks the code history, and honestly do not improve the code readability.